src/curl.c

```diff
@@ -363,6 +364,62 @@ int swupd_curl_get_file(const char *url, char *filename, struct file *file,
 	return err;
 }
 
+void swupd_curl_test_resume(void)
+{
+#define RESUME_BYTE_RANGE 2
+
+	CURLcode curl_ret;
+	char *version_string = NULL;
+	char *url = NULL;
+
+	if (!curl) {
+		abort();
+	}
```

Comment:
I would prefer to do "assert(curl)" here instead, since it indicates a programming error if curl is not initialized at this point in the code.

But, the code is not consistent in this regard, so maybe we should make this type of change throughout the code base at once in a separate commit.